addon/components/o-s-s/alert.ts

```diff
@@ -0,0 +1,38 @@
+import Component from '@glimmer/component';
+
+type skinType = 'success' | 'error' | 'info' | 'warning';
+
+interface OSSAlertArgs {
+  skin: skinType;
+  title?: string;
+  subtitle?: string;
+}
+
+export default class OSSAlert extends Component<OSSAlertArgs> {
+  constructor(owner: unknown, args: OSSAlertArgs) {
+    super(owner, args);
+
+    if (!args.skin) {
+      throw new Error('[component][OSS::Alert] The @skin argument is mandatory');
+    }
```

Comment:
how about setting 'info' as default ? 🤷

Reply:
You could set the default state of your switch below to 'fa-info-circle' & remove the `case 'info'` line ?

Reply:
Ok ok I make 'info' as default state